html/browsers/browsing-the-web/navigating-across-documents/about-srcdoc-navigation-blocked.window.js

```diff
@@ -0,0 +1,43 @@
+// META: title=Navigation to about:srcdoc, not via srcdoc="", must be blocked
+// META: script=../resources/helpers.js
+
+promise_test(async t => {
+  const iframe = await addSrcdocIframe();
+
+  iframe.contentWindow.location = "/common/blank.html";
+  await waitForIframeLoad(iframe);
+
+  iframe.contentWindow.location = "about:srcdoc";
+
+  // Fetching "about:srcdoc" should result in a network error, and navigating
+  // to a network error should produce an opaque-origin page. In particular,
+  // since the error page should end up being cross-origin to the parent
+  // frame, `contentDocument` should return `null`.
+  //
+  // If instead this results in a message because we reloaded the srcdoc iframe, immediately fail.
```

Comment:
I guess this is a bug we're seeing browsers, where the non-about:srcdoc page actually gets reloaded (and the parent frame gets a message)?

Reply:
I think the bug involves navigating to the srcdoc document that was created on line 5. I will tweak the wording to not use "reload".